scripts/kernel-arch-complete.sh

```diff
@@ -103,6 +103,20 @@ fi
         curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "sleep", "send_to": ["cip-dev+int+1680+2922216858170285886@lists.cip-project.org", "kernel-build-reports@lists.linaro.org", "kernelci-results@groups.io"], "format": ["txt"], "delay": 3600}' ${API}/send
         curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "usb", "send_to": ["cip-dev+int+1680+2922216858170285886@lists.cip-project.org", "kernel-build-reports@lists.linaro.org", "kernelci-results@groups.io"], "format": ["txt"], "delay": 3600}' ${API}/send
         curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "v4l2-compliance-uvc", "send_to": ["cip-dev+int+1680+2922216858170285886@lists.cip-project.org", "kernel-build-reports@lists.linaro.org", "kernelci-results@groups.io"], "format": ["txt"], "delay": 3600}' ${API}/send
+    elif [ "$TREE_NAME" == "cip-gitlab" ]; then
+        echo "Sending results for CIP (GitLab) tree"
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'", "build_report": 1, "format": ["txt"], "send_to": ["cip-dev+int+1680+2922216858170285886@lists.cip-project.org"], "delay": 60}' ${API}/send
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "baseline", "send_to": ["cip-dev+int+1680+2922216858170285886@lists.cip-project.org"], "format": ["txt"], "delay": 1800}' ${API}/send
```

Comment:
How about `baseline-cip-nfs`?

Reply:
Yea I thought about that too :P
It's also missing from the cip tree. Will add both in a new commit.

Reply:
There are lots of other omissions, and not just for CIP.
For example there aren't many ltp test plans being reported on. Is this intentional?

Reply:
Mostly intentional. I've just updated the CIP configurations for now.

Reply:
Thanks (as discussed on IRC).